pkgs/development/web/deno/default.nix

```diff
@@ -25,7 +26,16 @@ rustPlatform.buildRustPackage rec {
     rev = "v${version}";
     sha256 = "sha256-YDlZYYKibq8eAeGkU1qtTrJC0UF4YOK7XzkxWZjoQhM=";
   };
-  cargoSha256 = "sha256-3mXjAD4kzAaiFkK9XFEy7Df+ko6jHfWAle7gAoHxb7E=";
+
```

Comment:
If you're updating these to hash can `fetchFromGitHub` also use `hash`
And can you remove this space

Reply:
Absolutely, done!